elliott/elliottlib/util.py

```diff
@@ -548,5 +548,5 @@ def log(msg):
         component = labels['com.redhat.component']
         v = labels['version']
         r = labels['release']
-        all_payload_nvrs[component] = (v, r)
+        all_payload_nvrs.append((component, v, r))
```

Comment:
everywhere we were using this func, it was converted to a tuple - so return tuple directly